src/Schema/Events/Lists.ts

```diff
@@ -0,0 +1,115 @@
+import { OwnerType } from "../Values/OwnerType"
+import { ActionType } from "."
+
+/**
+ * Schema describing 'Lists' events
+ * @packageDocumentation
+ */
+
+/**
+ * When a user adds an artwork to an artwork list
```

Comment:
question: @abhitip the user can also delete artwork from the list. do we have to track the event for this?